docs-v2/content/en/docs/pipeline-stages/_index.md

```diff
@@ -29,6 +29,7 @@ will not push artifacts to a remote repository.
 | [Tag]({{< relref "/docs/pipeline-stages/taggers" >}}) | tag images based on different policies |
 | [Test]({{< relref "/docs/pipeline-stages/testers" >}}) | run tests with testers |
 | [Deploy]({{< relref "/docs/pipeline-stages/deployers" >}}) |  deploy with kubectl, kustomize or helm |
+| [Verify]({{< relref "/docs/pipeline-stages/verify" >}}) |  verify deployments with specified test containers |
```

Comment:
This path `/docs/pipeline-stages/verify` only exists in the `docs-v2/...` directory